deepfence_ui/app/scripts/helpers/auth-helper.js

```diff
@@ -51,17 +51,39 @@ export function isUserSessionActive() {
   return isSessionActive;
 }
 
-export function startTimerForRefreshToken() {
+export async function isUserSessionActiveAsync() {
+  let isSessionActive = false;
+  const authToken = localStorage.getItem('authToken');
+  if (authToken) {
+    setInterval(() => {
+      refreshAuthTokenIfRequired();
+    }, 60 * 5 * 1000);
+    await refreshAuthTokenIfRequired();
+  }
+  if (authToken) {
+    isSessionActive = true;
+  } else {
+    isSessionActive = false;
+  }
```

Comment:
can this be minified?
```suggestion
  if (authToken) {
    setInterval(() => {
      refreshAuthTokenIfRequired();
    }, 60 * 5 * 1000);
    await refreshAuthTokenIfRequired();
    isSessionActive = true;
  }
```